ios/browser/api/ads/brave_ads.mm

```diff
@@ -681,7 +681,7 @@ - (void)detailsForCurrentCycle:(void (^)(NSInteger adsReceived,
       nextPaymentDate =
           [NSDate dateWithTimeIntervalSince1970:list.next_payment_date];
     }
-    completion(list.ads_received_this_month, list.estimated_pending_rewards,
+    completion(list.ads_received_this_month, list.earnings_for_this_month,
```

Comment:
Based on the rest of the PR, this seems like the wrong variable, should be `earnings_this_month`, not `earnings_for_this_month`